SPECS/xapi.spec

```diff
@@ -1235,6 +1235,9 @@ Coverage files from unit tests
 %{?_cov_results_package}
 
 %changelog
+* Thu Feb 23 2023 Benjamin Reis <benjamin.reis@vates.fr> - 23.3.0-1.2
```

Comment:
Let update the date, because 23.3.0-1.1 was only built today, so your changelog entry would be older than the previous build.